src/backends/unix/system.c

```diff
@@ -107,6 +110,17 @@ Sys_Quit(void)
 void
 Sys_Init(void)
 {
+	char *term;
+
+	term = getenv("TERM");
+
+	if (!strcmp(term, "xterm-color") ||
+		!strcmp(term, "xterm-256color"))
+	{
+		color_active = true;
+
+		Sys_ConsoleOutput("\2Terminal supports colors.\n");
+	}
```

Comment:
The idea is good. However I would do it a little bit different, because checking only `xterm` only will break anything which is not xterm. For example multiplexers like screen or tmux or alternative terminals like alacritty or kitty. What about: Check if $TERM includes the substring `color` and if it does not check if $COLORTERM is set?

Reply:
Thank you!

I have updated code with COLORTERM and examples what it can be in case of xfce4-terminal.